data/testData.json

```diff
@@ -14,5 +14,6 @@
     "Rags and Riches": 4,
     "Tragedy": 5,
     "Comedy": 6
-  }
-}
\ No newline at end of file
+  },
+  "headerStory": "Two Cats And A LadyBug"
```

Comment:
Move "headerStory" to expected.json please.

Reply:
moved